app/components/Toolbar/presenter.jsx

```diff
@@ -97,6 +107,8 @@ Toolbar.propTypes = {
   positionFrom: React.PropTypes.number.isRequired,
   ntSequence: React.PropTypes.object,
   onPositionFromChange: React.PropTypes.func.isRequired,
+  onClearSelection: React.PropTypes.func.isRequired,
+  selections: React.PropTypes.array,
```

Comment:
what about having a `hasSelection` boolean prop instead? I do not think we will use selections in this component, so it might be better to only pass what is needed. WDYT?